src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +44,38 @@
      * Андрей - 600
      * София - 900</p>
      */
+
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
add access modifficator and make it as constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names